uber/templates/jobs/index.html

```diff
@@ -1,9 +1,27 @@
 {% extends "base-admin.html" %}
 {% block title %}Shifts{% endblock %}
+{% block page_styles %}
+<!--inside page_style -->
+<link rel="stylesheet" href="../static/styles/fullcalendar.min.css" />
+<link rel="stylesheet" href="../static/styles/fullcalendar.print.css" media="print"/>
```

Comment:
I did see a combined.js but wasn't sure what mechanism would be able to sync the contents from the static directories in the correct order. Once someone lets me know how that works, I'll go ahead and make sure the references aren't stored on a per page lvl